docs/concepts.rst

```diff
@@ -1465,3 +1465,13 @@ would not be scanned by Airflow at all. This improves efficiency of DAG finding)
 The scope of a ``.airflowignore`` file is the directory it is in plus all its subfolders.
 You can also prepare ``.airflowignore`` file for a subfolder in ``DAG_FOLDER`` and it
 would only be applicable for that subfolder.
+
+
+.pluginignore
+''''''''''''''
+
+A ``.pluginignore`` file specifies the directories or files in ``PLUGINS_FOLDER``
```

Comment:
Why the new file name? In my opinion, you can use `.airflowignore` and it will be easier to use.

Reply:
I agree with your opinion. 
>In my opinion, you can use .airflowignore and it will be easier to use.

It was fixed from `.pluginignore` to `.airflowignore` . :-)